pkg/simulation/config/serviceentry.go

```diff
@@ -0,0 +1,88 @@
+package config
+
+import (
+	networkingv1alpha3 "istio.io/api/networking/v1alpha3"
+	"istio.io/client-go/pkg/apis/networking/v1alpha3"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/howardjohn/pilot-load/pkg/simulation/model"
+)
+
+type ServiceEntrySpec struct {
+	App               string
+	AppType           model.AppType
+	Namespace         string
+	SEResolutionIndex int
+}
+
+type ServiceEntry struct {
+	Spec *ServiceEntrySpec
+}
+
+var _ model.Simulation = &ServiceEntry{}
+
+func NewServiceEntry(s ServiceEntrySpec) *ServiceEntry {
+	return &ServiceEntry{Spec: &s}
+}
+
+func (v *ServiceEntry) Refresh(ctx model.Context) error {
+	v.Spec.SEResolutionIndex = (v.Spec.SEResolutionIndex + 1) % 3
+	return v.Run(ctx)
+}
+
+func (v *ServiceEntry) Run(ctx model.Context) (err error) {
+	return ctx.Client.Apply(v.getServiceEntry())
+}
+
+func (v *ServiceEntry) Cleanup(ctx model.Context) error {
+	return ctx.Client.Delete(v.getServiceEntry())
+}
+
+func (v *ServiceEntry) getServiceEntry() *v1alpha3.ServiceEntry {
+
+	s := v.Spec
+	spec := networkingv1alpha3.ServiceEntry{}
+
+	var seResolution networkingv1alpha3.ServiceEntry_Resolution
+	switch s.SEResolutionIndex {
```

Comment:
networkingv1alpha3.ServiceEntry_Resolution(s.SEResolutionIndex) does the same without a big switch

Reply:
and covers NONE mode

Reply:
Done.